components/explore/SidebarFilter.vue

```diff
@@ -30,5 +30,6 @@ const open = computed(
   position: sticky;
   top: 72px;
   height: calc(100vh - 72px);
+  margin-right: 32px;
```

Comment:
?? out of scope

Reply:
ups looks like two PRs we married together (sending flowers and wedding cake)
It is from different PR from @prachi00

Reply:
my bad, I am closing this one, I raised another clean pr #5262